### ISSUE TITLE
bin/grav list not found?

### ISSUE THREAD

**Issue by sbsaylors** [linuxserver/docker-grav#25]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

root@8fa8f518ca14:/var/www/grav# grav list
bash: grav: command not found
root@8fa8f518ca14:/var/www/grav# bin/grav list
bash: bin/grav: No such file or directory
root@8fa8f518ca14:/var/www/grav# 

### Expected Behavior

dlinuxserver/grav:latest

### Steps To Reproduce

linuxserver/grav:latest

### Environment

```markdown
- OS: ubuntu 22.04
- How docker service was installed: apt-get install
```

### CPU architecture

x86-64

### Docker creation

```bash
version: '2.1'
services:
  grav:
    image: linuxserver/grav
    container_name: grav
    restart: unless-stopped
    environment:
      - DUID=1000
      - DGID=1000
      - GRAV_MULTISITE=subdirectory # optional
      - ROBOTS_DISALLOW=false # optional
    volumes:
      - /data/containers/grav/backup:/var/www/grav/backup
      - /data/containers/grav/logs:/var/www/grav/log
      - /data/containers/grav/user:/var/www/grav/user
    ports:
      - 1881:80
```

### Container logs

```bash
In short, GRAV default and admin page works. bin/GPM command doesn't - isn't found. Where is it in this install? Where can we use the bin/grav list (fails)

Looking at https://learn.getgrav.org/17/cli-console/grav-cli
```

**Comment by nemchik** [comment]
`cd /app/www/public && bin/grav list`

Also, the volumes you have in your compose are not correct for our images. Please check our readme.

P.s. our readme actually has the wrong command location for the grav install and bin folder. My answer here in this comment is correct, and I'll correct our readme tomorrow.

**Comment by sbsaylors** [comment]
Thanks nemchik!  Let me know the updated volumes, and I'll test it out! [as of 12/20/2022, readme shows below]

    volumes:
      - /path/to/appdata/config:/config

**Comment by nemchik** [comment]
That is correct for volumes. The only thing out of date in our readme is the docker command to access the grav cli.

We have the example:
```
docker exec -it -w /app/www/public/grav-admin grav bin/gpm
```

But the correct command is:
```
docker exec -it -w /app/www/public grav bin/gpm
```

**Comment by sbsaylors** [comment]
That worked! Thank you!

**Comment by nemchik** [comment]
Great!

https://github.com/linuxserver/docker-grav#application-setup (our readme) is now updated to reflect the correct command.
